py4cast/lightning.py

```diff
@@ -721,11 +722,28 @@ def on_validation_epoch_end(self):
             dict_metrics.update(self.psd_plot_metric.compute())
             dict_metrics.update(self.rmse_psd_plot_metric.compute())
             dict_metrics.update(self.acc_metric.compute())
+
+            mlflow_logger_idx = [
+                i for i, l in enumerate(self.loggers) if isinstance(l, MLFlowLogger)
+            ]
+            mlflow_logger = (
+                self.loggers[mlflow_logger_idx[0]] if mlflow_logger_idx else None
+            )
```

Comment:
```suggestion
            mlflow_logger = next(
                iter([o for o in self.loggers if isinstance(o, MLFlowLogger)]), None
            )
```
Why not putting that into a property or method of the lightining module ? It is duplicated twice already plus used in the tplots (and plots have access to the LM)

Reply:
Sounds an excellent idea !